g--- source/blender/imbuf/intern/oiio/openimageio_api.cpp.old	2017-12-27 14:50:11.890991255 +0300
+++ source/blender/imbuf/intern/oiio/openimageio_api.cpp	2017-12-27 14:50:32.071156237 +0300
@@ -213,7 +213,7 @@
 	in = ImageInput::create(filename);
 	if (!in) {
 		std::cerr << __func__ << ": ImageInput::create() failed:" << std::endl
-		          << OpenImageIO::geterror() << std::endl;
+		          << OIIO_NAMESPACE::geterror() << std::endl;
 		return NULL;
 	}
 
